thrifty-schema/src/main/java/com/microsoft/thrifty/schema/Field.java

```diff
@@ -54,6 +54,7 @@ private Field(Builder builder) {
         this.fieldNamingPolicy = builder.fieldNamingPolicy;
         this.annotations = builder.annotations;
         this.type = builder.type;
+        this.javaName = builder.javaName;
```

Comment:
Should we maybe mark it as transient while we're here for posterity or just leave it as is since it's going away?